src/main/java/gomedic/logic/parser/CliSyntax.java

```diff
@@ -5,13 +5,16 @@
  */
 public class CliSyntax {
 
-    /* Prefix definitions for a person */
+    /* General prefixes */
     public static final Prefix PREFIX_NAME = new Prefix("n/");
     public static final Prefix PREFIX_PHONE = new Prefix("p/");
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("ta/");
 
+    /* Prefix definitions for a doctor */
+    public static final Prefix PREFIX_DEPARTMENT = new Prefix("s/");
```

Comment:
Is it better to use `de/` instead, `/s` is not really intuitive

Reply:
yeap i think de/ would make more sense, shall update it!